tests/integration/test_rest_ingest_api/test.py

```diff
@@ -80,10 +90,15 @@ def setup_nodes():
 def test_ingest_api_basic_case(table, query, status):
     instance.ip_address = "localhost"
     # insert data
-    resp = instance.http_request(method="POST", url="dae/v1/ingest/default/tables/" + table, data=json.dumps(query))
+    resp = instance.http_request(method="POST", url="dae/v1/ingest/tables/" + table,
+                                 data=get_gzipped_data(query),
+                                 headers={
+                                     'content-type': 'application/json',
+                                     'content-encoding': 'gzip',
+                                 })
```

Comment:
now, the `content-encoding` is necessary, in other words, the `content` must be encoded (and encode-type maybe none), do I understand correctly?

Reply:
No，it is optional. If not set, it consider as no compression in reqeust body.

Reply:
no-compression case still exists in _test_poll_status_in_batch_case_